### PR TITLE
[SPARK-42285][DOC] Update Parquet data source doc on the timestamp_ntz inference option

### PR DESCRIPTION
**Pull request by gengliangwang**
### What changes were proposed in this pull request?

This is a follow-up of https://github.com/apache/spark/pull/39856. The configuration changes should be reflected in the Parquet data source doc 

### Why are the changes needed?

To fix doc
### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Preview:
image

### Was this patch authored or co-authored using generative AI tooling?

No

### CODE REVIEW

File: docs/sql-data-sources-parquet.md [hunk 1]
```diff
@@ -616,14 +616,15 @@ Configuration of Parquet can be done via `spark.conf.set` or by running
   <td>3.3.0</td>
 </tr>
 <tr>
-  <td><code>spark.sql.parquet.timestampNTZ.enabled</code></td>
```
Comment (dongjoon-hyun) [inline docs/sql-data-sources-parquet.md:619]: So, previous this was no-op, @gengliangwang ?

Comment (gengliangwang) [reply docs/sql-data-sources-parquet.md:619]: Yes

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Thank you!

**Comment by dongjoon-hyun** [comment]
BTW, did you upload the screenshot? The link seems to be broken, @gengliangwang .

Screenshot 2024-02-16 at 15 35 57

**Comment by gengliangwang** [comment]
@dongjoon-hyun Yes, the screenshot is just updated.

**Comment by gengliangwang** [comment]
Merging to master/branch-3.5. @dongjoon-hyun Thanks for the review.

**Pull request closed without merging**